sources/install.sh

```diff
@@ -2994,6 +3021,372 @@ function install_cowpatty() {
   add-test-command "cowpatty -V"
 }
 
+function install_yarn() {
+  colorecho "Installing yarn"
+  curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+  echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+  apt update
```

Comment:
iirc, `apt-get` should be used instead of `apt`when running things in script, @Dramelac can you confirm?